spec/features/user_links_an_artist_to_a_release_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+feature "user links an artist to a release", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.